intra_process_demo/src/image_pipeline/camera_node.cpp

```diff
@@ -21,7 +21,16 @@
 int main(int argc, char ** argv)
 {
   rclcpp::init(argc, argv);
-  auto camera_node = std::make_shared<CameraNode>("image");
-  rclcpp::spin(camera_node);
-  return 0;
+  try
+  {
+    auto camera_node = std::make_shared<CameraNode>("image");
+    rclcpp::spin(camera_node);
+    return 0;
```

Comment:
is there any best practice of keeping the try-catch as short as possible over forward declaring the pointer and then proof for `nullptr`?